.github/workflows/package.yaml

```diff
@@ -42,5 +42,11 @@ jobs:
           PROJECT: src/AppAny.Quartz.EntityFrameworkCore.Migrations.MySql/AppAny.Quartz.EntityFrameworkCore.Migrations.MySql.csproj
           VERSION: ${{ steps.release.outputs.tag }}
 
+      - name: Package | Publish | SQL
+        run: dotnet pack -c Release -o ./pack /p:Version=$VERSION $PROJECT
+        env:
+          PROJECT: src/AppAny.Quartz.EntityFrameworkCore.Migrations.SQL/AppAny.Quartz.EntityFrameworkCore.Migrations.SQL.csproj
```

Comment:
This project should be named like Microsoft EF Core package

![image](https://user-images.githubusercontent.com/46970457/207647548-522ef753-92aa-4c86-809b-565b7b64bddc.png)